modules/oe_subscriptions_anonymous/tests/src/FunctionalJavascript/SubscribeModalFormTest.php

```diff
@@ -115,6 +123,18 @@ public function testModalForm(): void {
     $this->assertSubscriptionCreateMailStatusMessage();
     $this->assertCount(1, $this->getMails());
     $this->assertMail('to', 'test@test.com');
+    $assert_session->addressEquals($node->toUrl());
+
+    // Test confirm message override.
+    $this->config('system.theme')->set('default', 'oe_subscriptions_anonymous_theme_test')->save();
```

Comment:
I appreciate a lot the effort on adding the tests, but I am not sure about this. It does end up checking that the page has been reloaded rather than using a command to render the messages...
Let's keep it because I like the idea of testing it, but let's add a big comment that explains the reason why this is there: the message command doesn't take into account template overrides etc etc.

Reply:
Done